src/content/developers/docs/data-availability.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Data availability 
+description: An overview of problems and solutions relating to data availability in Ethereum 
+lang: en
+sidebar: true
+---
+
+In Ethereum, a node proposing a new block is required to make the underlying data available to the remaining peer-to-peer nodes on the network. This concept, known as data availability, is fundamental to the security and functionality of every blockchain, including Ethereum. Data availability is so important that an entire field of research studying the so-called “data availability problem” exists. 
+
+## What is data availability? {#what-is-data-availability}
```

Comment:
All developer docs should have a prerequisites section. Ideally we have this in mind when writing/reviewing as it informs the level of granularity we should go into about specific topics—each developer doc isn’t a stand-alone article, ideally, it should live relative to every other developer doc with clearly defined expected knowledge to comprehend.

Reply:
Added this now.